mesoshttp/offers.py

```diff
@@ -25,7 +25,7 @@ def get_offer(self):
         '''
         return self.offer
 
-    def accept(self, operations):
+    def accept(self, operations, refuse_seconds=None):
```

Comment:
to be more generic, for more upcoming options, would be better to get something like:

    def accept(self, operations, options=None):
    ... and take refuse_seconds from options['filters']
        if options and options.get('filters'):
           message['accept']['filters'] = options.get('filters')